app/routes/users/components/UserProfile/index.tsx

```diff
@@ -123,7 +123,7 @@ const UserProfile = () => {
   }
 
   //If you wonder what this is, ask somebody
-  const FRAMEID = [6050, 5962, 7276, 7434, 7747, 8493];
+  const FRAMEID = [6050, 5962, 7276, 7325, 7434, 7747, 8493];
```

Comment:
🥺
```suggestion
  const FRAMEID = [6050, 5962, 7276,  8476, 7434, 7747, 8493];
```

Reply:
![image](https://github.com/user-attachments/assets/6045df34-fe06-4b89-83c0-4fc0bc3df93e)

Reply:
NOK 470, take it or leave it

Reply:
:no_entry_sign:

Reply:
😢